components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -39,6 +39,8 @@
 import org.wso2.carbon.identity.oauth2.model.RefreshTokenValidationDataDO;
 import org.wso2.carbon.identity.oauth2.util.OAuth2Util;
 import org.wso2.carbon.user.core.util.UserCoreUtil;
+import static org.wso2.carbon.identity.oauth2.util.OAuth2Util.buildScopeArray;
```

Comment:
If you are introducing this import, change every reference in this class with this.